library/UnicornHat/unicornhat.py

```diff
@@ -326,7 +326,7 @@ def set_pixel(x, y, r, g=None, b=None):
             r, g, b = COLORS[r.lower()]
         
         except KeyError:
-            raise ValueError('Invalid color!')
+            r, g, b = int(r,16).to_bytes(3,'big')
```

Comment:
This is a handy addition. But it would behave strangely with three char colours such as "FC0" or "FFF":

```python
>>> tuple(int("fff",16).to_bytes(3,'big'))
(0, 15, 255)
```

And fail confusingly with more than 6 chars:

```python
>>> int("FFFFffffffffffffdfFFFFFF",16).to_bytes(3,'big')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: int too big to convert
```

So I'd suggest something like:

```python
if len(r) == 6:
    r, g, b = int(r, 16).to_bytes(3, 'big')
else:
    raise ValueError("Invalid color!')
```